plugins/scaffolder-backend/src/scaffolder/tasks/NunjucksWorkflowRunner.ts

```diff
@@ -267,6 +246,38 @@ export class NunjucksWorkflowRunner implements WorkflowRunner {
               )) ??
             {};
 
+          if (task.isDryRun) {
+            taskLogger.info(
+              `Running ${
+                action.id
+              } in dry-run mode with inputs: ${JSON.stringify(
+                input,
+                undefined,
+                2,
+              )}`,
+            );
```

Comment:
I like it, but's a little bit scary because of the potential secret leakage. This might contain user tokens that won't be obfuscated if logged. I don't **think** there's any leakage atm, but it's a bit scary to leave this here for the future. What do you think, fine with skipping this?

Reply:
hm valid concern! However, logging out the action inputs makes the DX of dry run mode much better by being able to inspect the rendered inputs (eg. when using nunjucks filters) and prevent having to "live run" templates to catch certain bugs especially for actions that do not currently support dry running. 

I've updated the change to redact secrets from the logged inputs. Would that address the concern here?

Reply:
Yep, nice! Let's :shipit: